Dockerfile

```diff
@@ -19,6 +18,8 @@ FROM base as development
 ENV PORT=80
 EXPOSE ${PORT}
 
+ENV ENTITY_SERVICE_DEBUG=1
```

Comment:
why is debug enabled here and then turned off later? Should there be a comment?

Reply:
They pertain to two different build targets (development and production). So everything between "FROM base as development" to "FROM base as production" will only be run when building this image targeting `development` and everything after "FROM base as production" will only be run when building this image targeting `production`.

[Here's](https://docs.docker.com/engine/reference/commandline/build/#target) the docker documentation.

Reply:
Maybe add a small comment 'entering debug mode because....'